src/Router/index.js

```diff
@@ -0,0 +1,48 @@
+import { createStackNavigator, createAppContainer, createBottomTabNavigator } from 'react-navigation';
+
+import Todo from '../components/Todo';
+import DummyScreen from '../components/DummyScreen';
+
+import Routes from './routes';
+
+const tabNavigator = createBottomTabNavigator(
+  {
+    [Routes.Todo]: {
+      screen: Todo
+    },
+    [Routes.DummyScreen]: {
+      screen: DummyScreen
+    }
+  },
+  {
+    tabBarOptions: {
+      activeTintColor: 'red',
+      labelStyle: {
+        fontSize: 18
+      },
+      style: {
+        paddingBottom: 10
+      },
+      pressColor: 'blue'
+    },
+    navigationOptions: ({ navigation }) => ({
+      title: `${navigation.state.routes[navigation.state.index].routeName}`
+    })
+  }
+);
+
+const stackNavigator = createStackNavigator(
+  {
+    Home: tabNavigator
```

Comment:
Home is actually  a route name. You should added to the Routes object and use it like this `[Routes.Home]: `.

Reply:
Done on another branch called "bookList".